rules/jboss-eap/weblogic/XmlWebLogicRules.windup.xml

```diff
@@ -114,20 +114,20 @@
         </rule>
         <rule id="XmlWebLogicRules_14wscy">
             <when>
-                <xmlfile as="1" matches="/wl:weblogic-webservices | /wl9:weblogic-webservices">
+                <xmlfile as="webservices" matches="/wl:weblogic-webservices | /wl9:weblogic-webservices">
                     <namespace prefix="wl9" uri="http://www.bea.com/ns/weblogic/90"/>
                     <namespace prefix="wl" uri="http://www.bea.com/ns/weblogic/weblogic-webservices"/>
                 </xmlfile>
-                <xmlfile as="2" matches="//wl:webservice-type | //wl9:webservice-type" from="1">
+                <xmlfile as="webservices-type" matches="//wl:webservice-type | //wl9:webservice-type" from="webservices">
```

Comment:
Hey Sande,

Since this could also be plural, I suggest using plural "webservice-types" instead.

Thanks for this update!